IPs/RSKIP200.md

```diff
@@ -0,0 +1,65 @@
+# Bridge performance improvement
+
+|RSKIP          |200           |
+| :------------ |:-------------|
+|**Title**      |Receive headers limits |
+|**Created**    |08-JAN-21 |
+|**Author**     |PGP |
+|**Purpose**    |Sca,USa,Sec |
+|**Layer**      |Core |
+|**Complexity** |2 |
+|**Status**     |Draft |
+
+## Abstract
+
+This RSKIP proposes to include new functionality in the bridge precompiled contract to receive one header every time that this function is called and save this header in the internal storage of the bridge. 
+This also included a modification in the actual method to receive headers in the same precompiled contract. These methods receive an array of headers to be stored. The modification for this method implies not allow to be called for external users after this RSKIP. 
+
+## Motivation
+
+Bridge precompiled contract includes a function to can receive an array of heads to be stored. This function could be exposed to attacks. To avoid this, some new limits are included. 
+Powpeg nodes need to send a group of headers anyway. 
+Since one of these new limits is to receive only one header per time, and these limits are needed only for external users, a new method with the new limits is created and the original one was modified to be used only by powpeg nodes. 
+
+## Specification
+
+To implement this change we suggest making the following modifications:
+
+New function:
+```
+ reveive_header(bytes header) returns (int256 returnValue)
+```
+
+Modification in the actual receive_headers function:
+After this RSKIP the function needs to return to private. 
+
+
+### reveive_header limits
+
+1) The size of the parameter received is verified. These limits also exist in receive_headers function. 
+2) Header is not previously saved in storage.
+3) Time between calls to receive_headers is too short. The time specified between calls to this function is got using the constants of each network. The constant used to compare is named minSecondsBetweenCallsToReceiveHeader.
```

Comment:
The fact there is a time limit between calls should be included in the Specification maybe